react/components/ToastProvider/README.md

```diff
@@ -6,7 +6,7 @@
 
 ### 👎 Don'ts
 - Do not present critical or high priority actions on a Toast. If that's the case, you might consider using [Alerts](#alert) instead.
-- Due to its low to mild priority usage and dismissability, warning and error semantic styles do not apply to Toasts.
+- Due to its low to mild priority usage and dismiss ability, warning and error semantic styles do not apply to Toasts.
```

Comment:
I think that, as we have the word **dismissable**, we can have the word **dismissability** (as the quality to be dismissable). But it could be validated with the Localization team.

Reply:
Do you prefer that I wait for someone else's review before making this change?

Reply:
We can change it later 👍